assets/src/util/math.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * A helper function to help clarify code when clamping a value between a range.
+ * (Remove when tc39 implements `Math.clamp`)
+ *
+ * @returns {} {@link value} limited to the range defined by {@link min} and
+ * {@link max}
+ */
+export const clamp = (value: number, min: number, max: number): number =>
+  Math.min(Math.max(min, value), max)
```

Comment:
**suggestion:** would be good to also add a test

Reply:
I thought there was a [polyfill](https://www.npmjs.com/package/ecma-proposal-math-extensions) for this I could use but it seems like the repo for the polyfill is gone. 

I had thought I was going to use this function a lot more, hence pulling it out into it's own function, do you feel strongly on keeping this and adding tests vs getting rid of it and inlining?

Reply:
No real preference one way or another.

Reply:
a55dbf86bd6ba9b874d26258a2354cf67917a592